catalog/task/push-application-snapshot/0.2/README.md

```diff
@@ -0,0 +1,10 @@
+# push-snapshot
```

Comment:
I think we should change the directory name to `push-snapshot` and change the related filenames